packages/core/useWindowSize/index.ts

```diff
@@ -20,6 +20,12 @@ export interface UseWindowSizeOptions extends ConfigurableWindow {
    * @default true
    */
   includeScrollbar?: boolean
+
+  /**
+   * Whether to use the window.outerWidth and window.outerHeight instead of window.innerWidth and window.innerHeight
+   * @default false
+   */
+  useOuterSize?: boolean
```

Comment:
```suggestion
  type?: 'inner' | 'outer'
```

This would be more generic